ibr/ibr-sdk/build/ibr-sdk.js

```diff
@@ -187,25 +187,27 @@ typeof define === 'function' && define.amd ? define(['exports'], factory) :
     const ibrData = InternalBuildingRepresentation.read(
         new Pbf(ibrRawData));
     const ibrObject = new IBRObject( ibrData );
+    document.getElementById('dwn-btn').style.display = 'block';
     const structure = {};
-    // Visualization layers of current structure
-    structure['layers'] = renderLayer( ibrObject,
+    // Visualization visualizations of current structure
+    structure['visualizations'] = renderVisualization( ibrObject,
         structureIndex, scene );
     // Sub-structures of the current structure
     structure['structures'] = ibrObject.getSubStructures();
+    return saveToBinary( ibrObject );
```

Comment:
why is the render function saving to IBR?

Reply:
Separated them.